src/redux/slices/transactions/index.js

```diff
@@ -0,0 +1,26 @@
+import { createSelector } from 'reselect';
+
+const SLICE_NAME = 'transactions';
+
+// A helper function to create the parameter selectors
```

Comment:
We'll be using this for a _lot_ of selectors; we'll need it in every file we need parameterized selectors.  Would be nice to get this into `src/redux/createParameterSelector.js` so we can import it here and in the other slices that use it rather than duplicate the function in each file; can you also update any other slices that contain this function to import it also? I think it's a small enough change that we can put that extraction + updating the references in other slice(s) to use it in this PR comfortably since it is not a behavioural change, just structural at the filesystem level really.